components/inputs/docs/input-radio.md

```diff
@@ -66,10 +99,28 @@ class MyElem extends LitElement {
 }
 ```
 
-## Radio Spacer
+The example below uses a component similar to that in the code above which allows toggling of properties in order to see what those styles look like.
+
+<!-- docs: demo live name:d2l-test-input-radio-solo display:block -->
+```html
+<script type="module">
+  import '@brightspace-ui/core/components/inputs/demo/input-radio-solo-test.js';
+</script>
+<d2l-test-input-radio-solo></d2l-test-input-radio-solo>
+```
+
+## Radio Spacer [d2l-input-radio-spacer]
 
 To align related content below radio buttons, the `d2l-input-radio-spacer` element can be used:
 
+<!-- docs: demo display:block -->
```

Comment:
Why do these need `display: block`?

Reply:
I feel like there was some weird spacing happening, but yeah they do seem fine. I'll re-test and remove.

Reply:
Yeah no problem to leave it, just expected these to work well with `display: flex`